agent/src/main/java/io/jenkins/plugins/remotingkafka/Options.java

```diff
@@ -0,0 +1,18 @@
+package io.jenkins.plugins.remotingkafka;
+
+import org.kohsuke.args4j.Option;
+
+public class Options {
+
+    @Option(name = "-name", usage = "Name of the slave")
+    public String name;
+
+    @Option(name = "-master", usage = "The complete target Jenkins URL like 'http://server:8080/jenkins/'.")
+    public String master;
+
+    @Option(name = "-help", aliases = "--help", usage = "Show the help screen")
+    public boolean help;
+
+    @Option(name = "-kafkaURL", usage = "Kafka host and port address identifier")
+    public String kafkaURL;
+}
```

Comment:
It would be great to have some security for such connection mode, e.g. like passing secrets in common Remoting agent CLI. I will create a follow-up ticket, IMHO fine for now